public/video-ui/styles/components/_forms.scss

```diff
@@ -137,6 +137,9 @@
 
   }
 
+  &--textarea {
```

Comment:
we set height to 36px [above](https://github.com/guardian/media-atom-maker/pull/746/files#diff-b745c7e774cc24d5f1eb2621797259efR73)... rather than creating a new class for the new form field, just use css specificity 😄